config/locales/alchemy.it.yml

```diff
@@ -0,0 +1,924 @@
+    "Welcome to Alchemy": "Benvenuto su Alchemy"
+    "Who else is online": "Chi altro è online"
+    "Yes": "Sì"
+    "You are not authorized": "Non sei autorizzato"
+    "You are about to edit many pictures at once": "Stai per modificare %{length} immagini contemporaneamente"
+    element_dirty_close_window_notice: "Hai degli elementi non salvati. Vuoi veramente chiudere la finestra degli elementi?"
+    "You may upload files with following extensions": "Puoi caricare %{file_types_description} con le seguenti estensioni: %{file_types}"
+    "You may upload any file": "Puoi caricare qualsiasi file"
+    "Your last login was on": "Il tuo ultimo login è stato alle %{time}."
+    "Your last updated pages": "Le tue pagine modificate di recente"
+    "Your trash is empty": "Il cestino è vuoto"
+    above: "sopra"
+    add: "Aggiungi"
+    "add new content": "Aggiungi nuovo contenuto"
+    add_image_to_element: "Aggiungi un immagine"
+    align_in_text: "Allineamento nel testo"
+    all: "tutto"
+    all_pictures: "Tutte le immagini"
+    apply: "applica"
+    assign_file: "Assegna un file"
+    assign_file_from_archive: "assegna un file dal tuo archivio"
+    assign_image: "Assegna un'immagine"
+    attachment_filename_notice: "* Prego non utilizzare alcun carattere speciale per il nome del file."
+    auto_play: "Riproduci il video dopo il caricamento"
+    big_thumbnails: "Miniature grandi"
+    browse: "sfoglia"
+    cancel: "annulla"
+    cannot_delete_picture_notice: "Impossibile eliminare l'Immagine %{name} poiché è ancora in uso."
+    choose_element_as_target: "Prego scegli un elemento come destinatario"
+    choose_element_to_link: "Prego scegli un elemento"
+    choose_file_to_link: "Prego scegli un file da linkare"
+    "clear clipboard": "svuota appunti"
+    "clear trash": "svuota cestino"
+    click_to_show_all: "Clicca per tornare a mostrare tutto."
+    confirm_to_delete_file: "Vuoi veramente eliminare questo file dal server?"
+    confirm_to_delete_image: "Vuoi veramente eliminare questa immagine dal server?"
+    confirm_to_delete_image_from_server: "Vuoi veramente eliminare questa immagine dal server?"
+    confirm_to_delete_images_from_server: "Vuoi veramente eliminare queste immagini dal server?"
+    confirm_to_delete_page: "Vuoi eliminare veramente questa pagina? I suoi elementi andranno persi!"
+    content_essence_not_found: "Essenza del contenuto non trovata"
+    content_not_found: "Campo per il contenuto non trovato."
+    content_validations_headline: "Prego controlla i campi segnati qui sotto"
+    copy: "copia"
+    copy_element: "Copia questo elemento"
+    copy_page: "Copia pagina"
+    "Could not delete Pictures": "Impossibile eliminare le Immagini"
+    copy_language_tree_heading: "Copia albero delle pagine"
+    country_code_placeholder: 'ad es. us (opzionale)'
+    country_code_foot_note: "Basta che imposti un codice nazione se vuoi fornire ogni nazione con una lingua individuale."
```

Comment:
`Basta impostare un codice nazione per dotarla di una lingua individuale` but the meaning is not super clear to me, even in [english](https://github.com/AlchemyCMS/alchemy_cms/blob/cb3cd5ad4ffeffb15ed6b561872901146cd7e1b6/config/locales/alchemy.en.yml#L333)

@tvdeyen can you help to clarify this one as well please?